docs/integrations/sources/my-hours.md

```diff
@@ -22,7 +22,11 @@ This source allows you to synchronize the following data tables:
 ## Getting started
 
 **Requirements**
-In order to use the My Hours API you need to provide the credentials to an admin My Hours account.
+
+ 1. In order to use the My Hours API you need to provide the credentials from My Hours account.
```

Comment:
The connector currently does this behind the scene. I think we should keep this behavior

Reply:
A little more context on why this matter: The access token returned by the endpoint is only valid for 5 days
```
{"accessToken":"<access_key>","refreshToken":"<refresh_token>","expiresIn":432000}
```